src/Elliptic/DiscontinuousGalerkin/DgOperator.hpp

```diff
@@ -273,6 +277,8 @@ struct DgOperatorImpl<System, Linearized, tmpl::list<PrimalFields...>,
       const std::tuple<FluxesArgs...>& fluxes_args,
       const std::tuple<SourcesArgs...>& sources_args,
       const DirectionMap<Dim, std::tuple<FluxesArgs...>>& fluxes_args_on_faces,
+      const std::function<bool(const ElementId<Dim>&)>& data_is_zero =
```

Comment:
I haven't looked at any of the code in detail. Note that `std::function` is pretty slow compared to lambdas or a template parameter. If the  `std::function` is needed in a loop you might want to think about an alternative way of handling this. Basically, `std::function` does type erasure internally, which is why it's slower

Reply:
I think making the argument type a template argument will work without having to change anything else.  (And all this appears again below.)

Reply:
Changed in a fixup, please take a look